chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -951,6 +951,7 @@ public ActivityMetadataPage metadataForPendingSurveillances(@RequestParam(requir
         return activityManager.getApiKeyActivity(startDate, endDate);
     }
 
+    @Deprecated
```

Comment:
Probably means we need an MR to update the api_security table to add "DEPRECATED" as a note to this endpoint. Might need to add all of these /metadata/beta endpoints too